community/Gretl.gitignore

```diff
@@ -0,0 +1,11 @@
+# gitignore template for gretl
```

Comment:
Missed capitalization?
```suggestion
# gitignore template for Gretl
```

Reply:
Actually, the software is named "gretl" but I named the file ```Gretl.gitignore``` as all existing file names I saw started with y capital letter.
But I am fine with using the capital letter "G" for "Gretl"